src/lib/tree/outlet.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  Directive,
+  ViewContainerRef,
+} from '@angular/core';
+
+/**
+ * Outlet for nested CdkNode. Put `[matTreeNodeOutlet]` on a tag to place children dataNodes
+ * inside the outlet.
+ */
+@Directive({
+  selector: '[matTreeNodeOutlet]'
+})
+export class MatTreeNodeOutlet {
```

Comment:
Should add here `implements CdkTreeNodeOutlet` to make sure they stay the same API